src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -24,7 +24,12 @@ <h2 class="addon"{{ addon.name|locale_html }} id="addon-name">
       <span class="version">{{ version.version }} ({{ version.get_channel_display() }})</span>
     {% endif %}
   </h2>
-  <h4 class="author">by {{ users_list(addon.listed_authors) }}</h4>
+  <h4 class="author">by
+    {% for author in addon.listed_authors %}
+    <a href="{{ url('reviewers.developer_profile', author.id) }}">{{ author.name }}</a>
```

Comment:
Does this render the same thing as `users_list`? I.e., do we need some separators or any markup between the authors?

Reply:
Also, do you think we should have a test for this (I don't see one added)?

Reply:
mmm, I think I'm missing the `, `.  I'll add.